### DISCUSSION TITLE
How do I get rid of thus color overlay for folders in wsl

### DISCUSSION THREAD

**Discussion by RiteshChepuri** [microsoft/WSL#11954] [Q&A]
![Screenshot 2024-08-24 142108](https://github.com/user-attachments/assets/6e70545f-1606-4c91-b181-abb231bee143)

**Comment by paul-haskell** [comment]
What tool or program are you running to get your folder list?  Is this the output from 'ls' in a terminal window?

**Reply by RiteshChepuri** [reply]
Yeah, I use lsd with alias ls = lsd. It only happens to the folders in windows system not in the wsl ecosystem

**Comment by paul-haskell** [comment]
'ls' can be configured to print different types of files and directories in different colors.  My guess is that is what is happening in your case.

- Try running "alias lsd".  I bet "lsd" is the plain vanilla "ls" with a bunch of options that include color display.
- Try running "alias ls=/usr/bin/ls" and then run "ls".  Do all the colors go away?
If so, and you don't want the colors, then you have a solution!  You can just alias "ls" to include only the options you want, and not the options you don't want, including color display.  If you do like the colors but don't like these particular colors, then please read [LINK](https://www.bigsoft.co.uk/blog/2008/04/11/configuring-ls_colors) for instructions on how to configure colors the way you want.

(I believe that on your system, the Windows folders and the WSL folders have different permissions, so they are colored differently by "ls".  I just fought through that problem today also.)

**Comment by RiteshChepuri** [comment]
yeah lsd is something that list with the icons for files and folders. with color formatting for permissions and stuff. lsd is the one thats causing that colors problems. I removed my alias and it works fine but without the icons and colors. 
Thank you
